Makefile

```diff
@@ -75,6 +75,10 @@ ifneq ($(HAS_SERVER),)
 	cd server && env GOOS=linux GOARCH=amd64 $(GO) build -o dist/plugin-linux-amd64;
 	cd server && env GOOS=darwin GOARCH=amd64 $(GO) build -o dist/plugin-darwin-amd64;
 	cd server && env GOOS=windows GOARCH=amd64 $(GO) build -o dist/plugin-windows-amd64.exe;
+ifneq ($(HAS_PUBLIC),)
+	mkdir -p server/dist/public;
```

Comment:
What do you think about pre creating this, similar to `assets` https://github.com/mattermost/mattermost-plugin-sample/tree/master/assets?

Reply:
Yeah, especially with getting rid of the static_files setting in the plugin.json. It makes sense.